content/sensu-enterprise/3.2/guides/troubleshooting.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "Troubleshooting Sensu Enterprise"
+linkTitle: "Troubleshooting"
+product: "Sensu Enterprise"
+version: "3.2"
+weight: 10
+menu:
+ sensu-enterprise-3.2:
+   parent: guides
+---
+
+This guide provides an introduction to troubleshooting Sensu Enterprise.
+For information about troubleshooting the Sensu client and RabbitMQ connection, see the [Sensu Core troubleshooting guide][1].
+
+- [Reloading Configuration](#reloading-configuration)
+- [Collecting Logs](#collecting-logs)
+- [Settings Log Levels](#setting-log-levels)
+- [Printing Configuration](#printing-configuration)
+- [Resolving Private Key Errors](#resolving-private-key-erors)
+
+### Reloading Configuration
+
+Following any changes to the Sensu Enterprise configuration, you must reload Sensu Enterprise in order for the changes to take effect.
+
+Reload Sensu Enterprise on Linux distributions using `systemd`:
+
+{{< highlight shell >}}
+sudo systemctl reload sensu-enterprise{{< /highlight >}}
+
+Reload Sensu Enterprise on Linux distributions using `sysvinit`:
+
+{{< highlight shell >}}
+sudo service sensu-enterprise reload{{< /highlight >}}
+
+When making changes to the Sensu client configuration, restart the client using the process described in the [Sensu Core troubleshooting guide][2].
+
+### Collecting Logs
+
+Sensu's logs provide a wealth of information when troubleshooting issues. They live at `/var/log/sensu`:
+
+{{< highlight shell >}}
+/var/log/sensu
+├── sensu-enterprise.log
+└── sensu-enterprise-dashboard.log{{< /highlight >}}
+
+You can view the Sensu Enterprise logs at the paths above, or provide them in an archive:
+
+{{< highlight shell >}}
+tail -n 10000 /var/log/sensu/sensu-enterprise.log > sensu-enterprise-10k.log && gzip -9 sensu-enterprise-10k.log{{< /highlight >}}
```

Comment:
maybe call out above that this is only the last 10000 lines?

Reply:
Fixed in https://github.com/sensu/sensu-docs/pull/790/commits/6ed9b0d7f943a2064ddaa89891e48531b59977f8